src/components/Web3Status/ConnectWalletPopover.tsx

```diff
@@ -170,8 +189,14 @@ export const ConnectWalletPopover = ({ setModal, tryActivation, children }: Conn
     <Wrapper>
       <StyledPopover
         innerRef={popoverRef}
-        content={<List>{getOptions()}</List>}
+        content={
+          <List>
+            {getOptions()}
+            { active && <Disconnect onClick={disconnect}></Disconnect> }
```

Comment:
Would it maybe make more sense to put it inside the getOptions instead of handling it like this? I haven't digged that much but if it can be done easily. Also I haven't looked at designs but should it not show if we  are not connected? Or should it be disabled?

Reply:
it shouldn't show if user is not connected. Putting it inside `getOptions` would require extra conditionals and duplicate for mobile and web display. I definitely think this is a better option.